lib/modules/datasource/index.ts

```diff
@@ -278,6 +278,7 @@ async function fetchReleases(
   const datasource = getDatasourceFor(datasourceName);
   // istanbul ignore if: needs test
   if (!datasource) {
+    logger.warn('Unknown datasource: ' + datasourceName);
```

Comment:
```suggestion
    logger.warn({ datasource: datasourceName }, 'Unknown datasource');
```

We want to avoid string interpolation in Errors and Warnings